scripts/check_urls/check_urls.py

```diff
@@ -223,24 +223,14 @@ def send_slack_message(invalid_uris_list: List[IOUrl]):
     urls_black_list = {
         # 403 when this check runs (in the middle of the night)
         "https://id.lepida.it/docs/manuale_utente.pdf",
-        # still not available
-        "https://io.italia.it/carta-giovani-nazionale/informativa-beneficiari",
-        # still not available
-        "https://io.italia.it/carta-giovani-nazionale/guida-beneficiari",
-        # the article is available but the response status code is 403
-        "https://help.mixpanel.com/hc/en-us/articles/115004494803-Disable-Geolocation-Collection",
-        "https://assets.cdn.io.italia.it",
-        "https://www.trusttechnologies.it/wp-content/uploads/SPIDPRIN.TT_.DPMU15000.03-Guida-Utente-al-servizio-TIM-ID.pdf",
-        "https://www.trusttechnologies.it/contatti/#form",
-        "https://support.namirial.com/it/faq/faq-tsp/faq-tsp-spid",
-        "https://paytipper.com/wp-content/uploads/2021/02/logo.png",
-        "https://fims-dev-app-provider.azurewebsites.net/",
-        # returns a 404 anytime the check runs but it actually works fine
+         # returns a 404 anytime the check runs but it actually works fine
         "https://checkout.pagopa.it/dona",
-        # todo remove from blacklist when the url is available
-        "https://io.italia.it/app-content/tos_privacy_cartacultura.html/",
-        # todo remove from blacklist when the url is available
-        "https://cittadini.notifichedigitali.it"
+        # localhost is not reachable
+        "https://localhost",
+        # local url used for E2E tests
+        "http://127.0.0.1:3000/api/v1/cgn/delete",
+        # not accessible outside of PagoPA
+        "https://docs.google.com/presentation/d/11rEttb7lJYlRqgFpl4QopyjFmjt2Q0K8uis6JhAQaCw/edit#slide=id.p"
```

Comment:
```suggestion
        # Mixpanel EU endpoint
        "https://api-eu.mixpanel.com"
        # Profile 412 status error types
        # Already taken type
        "https://ioapp.it/problems/email-already-taken"
        # EIC UAT endpoint
        "https://collaudo.idserver.servizicie.interno.gov.it/idp"
        # localhost is not reachable
        "https://localhost",
        # local url used for E2E tests
        "http://127.0.0.1:3000/api/v1/cgn/delete",
        # not accessible outside of PagoPA
        "https://docs.google.com/presentation/d/11rEttb7lJYlRqgFpl4QopyjFmjt2Q0K8uis6JhAQaCw/edit#slide=id.p"
```

[Internal reference](https://pagopa.atlassian.net/wiki/spaces/IAEI/pages/1007910936/Broken+Links+su+AppIO).

What do you think if we address in this PR even the `https://assets.cdn.io.italia.it/logos/organizations/` case you can read about in the table in the internal reference above?

I remember what we said about the mixpanel endpoint, but what about putting it in blacklist even if in the future we'll move it in inside an `.env.*` files?

Reply:
Makes sense, thanks. Addressed in [d3e6a8d](https://github.com/pagopa/io-app/pull/5662/commits/d3e6a8ddb893826c516efaf5cd114758716dd7f7).